prow/config/config.go

```diff
@@ -202,6 +202,11 @@ type Plank struct {
 	// PodPendingTimeout is after how long the controller will perform a garbage
 	// collection on pending pods. Defaults to one day.
 	PodPendingTimeout time.Duration `json:"-"`
+	// PodRunningTimeoutString compiles into PodRunningTimeout at load time.
```

Comment:
I know you copied the above but I think we want the useful GoDoc (description, default) on this line so that people looking for options in godoc will know what they are looking for

Reply:
Can we use `metav1.Duration` or the custom duration type we use for DecorationConfig instead of continuing this two field pattern?

Reply:
ack

Reply:
How'd metav1.Duration handles default?

Reply:
It doesn't, we still need defaulting logic to set the value to something reasonable if the value parsed from config is 0 like you have now. The benefit of metav1.Duration is that it properly handles the parsing of duration strings so we don't need both a string field and a duration field.

Reply:
(I tried to change it to a pointer which can handle defaults better, see https://github.com/kubernetes/test-infra/pull/12607)

Reply:
Crap, already forgot you built that type. A pointer with `omitempty` SGTM